### DISCUSSION TITLE
No "Console Only" option in UTM 4.x

### DISCUSSION THREAD

**Discussion by nzedler** [utmapp/UTM#4533] [UTM Help (macOS)] [answered]
I downloaded the latest UTM version 4.0.8, however the "Console Only" option is missing in the display settings. This is required when installing Kali Linux (as stated in the official documentation as well: [Kali Doc](https://www.kali.org/docs/virtualization/install-utm-guest-vm/)), otherwise the screen stays black during the installer. 
When using UTM 3.2.4 from the Github Releases, everything works as expected. Using the latest UTM 4.x version is impossible like this.
I'm creating an issue, because this option is only partially related to Kali Linux.

**Comment by osy** [comment, marked as answer]
https://docs.getutm.app/guides/kali/#black-screen-on-start

The kali site is a wiki, someone should edit it.

**Reply by nzedler** [reply]
So if this is due to poor documentation, how do I create a terminal device in UTM 4.x, so that I can install a distro like Kali?

**Reply by osy** [reply]
Please click the link that you responded to.

**Reply by ivaquero** [reply]
Maybe my question sounds stupid, the link says "create a new serial device.", then what is a serial device? USB?

**Reply by osy** [reply]
Did you click that link as well?

**Reply by ivaquero** [reply]
Yeah, I clicked the link. It says "Open the VM settings and create a new serial device.". 
I am not sure what a serial device is.

**Reply by osy** [reply]
> To add a new device, click “New…” on the sidebar under “Devices” and select the device type. To remove a device, right/secondary click the device on the sidebar and choose “Remove”.

Then click "Serial"

**Reply by ivaquero** [reply]
Thank you @osy! I just found my mistake.

**Reply by damjanned** [reply]
Where is New.. under Devices?? I don't have that option, anyone knows why?

<img width="493" alt="Screenshot 2023-04-19 at 3 07 11 pm" src="https://user-images.githubusercontent.com/25449925/232972141-14f24793-15d6-42d9-b16f-68ebba8fee11.png">

**Reply by osy** [reply]
You’re probably on an old version.

**Reply by damjanned** [reply]
I found it and was able to install it. First while installation I had to have it as Console, once it's installed, remove the iso disk so it doesn't boot into the installation again, then come to this option and change it back to Full Graphics. See SS below:
<img width="895" alt="Screenshot 2023-04-20 at 11 21 43 am" src="https://user-images.githubusercontent.com/25449925/233233627-da73ac6e-d2ec-481f-82c0-2b8f9152a754.png">
